src/main/kotlin/com/devfloor/untitled/articlehitscount/application/ArticleHitsCountService.kt

```diff
@@ -0,0 +1,22 @@
+package com.devfloor.untitled.articlehitscount.application
+
+import com.devfloor.untitled.article.domain.Article
+import com.devfloor.untitled.articlehitscount.domain.ArticleHitsCount
+import com.devfloor.untitled.articlehitscount.domain.ArticleHitsCountRepository
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+
+@Service
+class ArticleHitsCountService(
+    private val articleHitsCountRepository: ArticleHitsCountRepository,
+) {
+    @Transactional
+    fun increase(article: Article) {
+        var articleHitsCount = articleHitsCountRepository.findByArticleOrNull(article)
+        if (articleHitsCount == null) {
+            articleHitsCount = ArticleHitsCount(article)
+        }
```

Comment:
여기 엘비스 연산자를 이용하는건 어떨까요 !?
```suggestion
       var articleHitsCount = articleHitsCountRepository.findByArticleOrNull(article)
            ?: ArticleHitsCount(article)
```

Reply:
수정했습니다!